tests/testthat/data_helper.R

```diff
@@ -15,3 +16,31 @@ simple_pred_data <- function() {
                         truth = as.factor(c(1,1,2,2,1,1,2,1,2,2,2,1,1,1,2,1)),
                         response = as.factor(c(1,2,1,2,1,2,1,2,1,2,1,2,1,2,1,2)))
 }
+
+adult_benchmark_result <- function(){
+  design = benchmark_grid(
+    tasks = tsk("adult_train"),
+    learners = lrns(c("classif.ranger", "classif.rpart"),
+                    predict_type = "prob", predict_sets = c("train", "test")),
+    resamplings = rsmps("cv", folds = 3)
+  )
+
+  return(benchmark(design))
+}
+
+adult_resample_result <- function(){
+  task = tsk("adult_train")
```

Comment:
use only a few lines of the data, this should be fast

Reply:
Same as above